.idea/copyright/pileproject.xml

```diff
@@ -2,6 +2,6 @@
   <copyright>
     <option name="allowReplaceKeyword" value="" />
     <option name="myName" value="pileproject" />
-    <option name="notice" value="Copyright (C) 2011-2015 PILE Project, Inc. &lt;dev@pileproject.com&gt;&#10;&#10;Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);&#10;you may not use this file except in compliance with the License.&#10;You may obtain a copy of the License at&#10;&#10;     http://www.apache.org/licenses/LICENSE-2.0&#10; &#10;Unless required by applicable law or agreed to in writing, software&#10;distributed under the License is distributed on an &quot;AS IS&quot; BASIS,&#10;WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.&#10;See the License for the specific language governing permissions and&#10;limitations under the License." />
+    <option name="notice" value="Copyright (C) 2011-2016 PILE Project, Inc. &lt;dev@pileproject.com&gt;&#10;&#10;Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);&#10;you may not use this file except in compliance with the License.&#10;You may obtain a copy of the License at&#10;&#10;     http://www.apache.org/licenses/LICENSE-2.0&#10; &#10;Unless required by applicable law or agreed to in writing, software&#10;distributed under the License is distributed on an &quot;AS IS&quot; BASIS,&#10;WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.&#10;See the License for the specific language governing permissions and&#10;limitations under the License." />
```

Comment:
Oh, I forgot to show a solution I thought. For example, the Dagger project uses `The Dagger Authors` and I think this can also apply our products something like `The Drive Authors`.